ReactiveObjCTests/NSObjectRACSelectorSignalSpec.m

```diff
@@ -194,6 +196,42 @@ - (id)objectValue;
 		expect(@([object respondsToSelector:selector])).to(beTruthy());
 	});
 
+	qck_it(@"hotpatch with dynamic subclass: disable a selector then re-implementing it in forwardInvocation:", ^{
+		__block BOOL patchApplied = NO;
+		
+		RACTestObject *object = [[RACTestObject alloc] init];
+		
+		// A dynamic subclass created.
+		[[RACObserve(object, objectValue) publish] connect];
+		[object rac_signalForSelector:@selector(lifeIsGood:)];
+		
+		// Simulator of hotpatch:
+		SEL selectorPatched = @selector(methodHotpatch);
```

Comment:
Oops, I missed it.
The short answer is NO, currently.

Hot-patch implements `forwardInvocation:` on original class. But the calling process of `lifeIsGood:` will be intercepted by `RACForwardInvocation` on `dynamic subclass`:

``` objc
BOOL matched = RACForwardInvocation(self, invocation);
if (matched) return;
```

Good news is, it can be identified by checking (`aliasSelector on dynamic subclass` != `originalSelector on original class`) to know whether it has been hot-patched (same situation with method swizzle) or not.

One more step forward, we can just checking (`originalSelector on original class` == `_objc_msgForward`) to know it. But I don't recommend it.